jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/WebServerControlRunSteps.java

```diff
@@ -60,13 +81,14 @@ public void verifyIfDeleteWebServerIsSuccessful(final String webServerName) {
     }
 
     @Then("^I see the httpd.conf$")
-    public void verifyProperConfFile() {
+    public boolean verifyProperConfFile() throws TimeoutException{
```

Comment:
If this method is no longer used in any other feature we should make it private and remove the @Then annotation